### DISCUSSION TITLE
Dockview and GridView json compatibility

### DISCUSSION THREAD

**Discussion by husayt** [mathuo/dockview#399] [Q&A]
Hi, Thank you for such a useful library. I am currently experimenting with it to see how we can use it. I have a question, as trying to use Docview as a way to set up panel layouts, and then use Gridview to display these layouts. Dockview is a thin layer over Gridview, I expect, json exported from Dockview to be accepted and compatible with GridView. But Gridview throws error when trying to pass it Dockview json. Is there a way perhaps to have a version of fromJson from DockView which is compatible with GridView?

**Comment by techbech** [comment]
I don't think it's impossible to make, but why? Why not just use the gridview json for gridviews?

**Reply by husayt** [reply]
Very simple use case. I need a simple gridview layout (i.e. without tabs) for the standard screen, but also I need to provide edit mode for user to update the layout visually. So the idea we use dockview for editmode with drag/drop. Once user saves the layout we want to load that layout and use with gridview.

**Reply by Code-Victor** [reply]
I have the same use case. What did you eventually settle on @husayt
